core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -127,8 +109,8 @@ private[data] sealed abstract class WriterTInstances1 extends WriterTInstances2
 }
 
 private[data] sealed abstract class WriterTInstances2 extends WriterTInstances3 {
-  implicit def catsDataMonadWriterForWriterT[F[_], L](implicit F: Monad[F], L: Monoid[L]): MonadWriter[WriterT[F, L, ?], L] =
-    new WriterTMonadWriter[F, L] {
+  implicit def catsDataMonadForWriterT[F[_], L](implicit F: Monad[F], L: Monoid[L]): Monad[WriterT[F, L, ?]] =
```

Comment:
This can be moved to `WriterTInstances0`, which means that the `Alternative`, `Applicative`, `FlatMap`, ... instances can be moved up as well.